xml/s4s_components.xml

```diff
@@ -412,32 +399,29 @@
     </listitem>
    </itemizedlist>
    <important>
-    <title>
-     Packages May Be Missing from Patterns
-    </title>
+    <title>Packages may be missing from patterns</title>
     <para>
      The selection of packages of the software patterns is defined while a
      specific release (Service Pack or major version) of &sles4sap; is
      developed. This package selection is stable over the lifetime of this
      particular release. When working with &sap; applications that have been
-     released more recently than your &sles4sap; version,
-     dependencies can be missing from the patterns.
+     released more recently than your &sles4sap; version, dependencies can be
+     missing from the patterns.
     </para>
     <para>
-     For definitive information about the dependencies of your &sap; application,
-     see the documentation provided to you by &sap;.
+     For definitive information about the dependencies of your &sap;
+     application, see the documentation provided to you by &sap;.
     </para>
    </important>
   </sect3>
-
   <sect3 xml:id="sec-component-clustertool">
    <title><systemitem>ClusterTools2</systemitem></title>
    <para>
-    <systemitem>ClusterTools2</systemitem> provides tools that help set up
-    and manage a Corosync/<systemitem>pacemaker</systemitem> cluster. Among
-    them are <systemitem>wow</systemitem> which helps create
-    highly available system resources, and
-    <systemitem>ClusterService</systemitem> which allows managing a cluster.
+    <systemitem>ClusterTools2</systemitem> provides tools that help set up and
+    manage a Corosync/<systemitem>pacemaker</systemitem> cluster. Among them
```

Comment:
Maybe it's better to use the entities here:

```suggestion
    manage a &coro; &amp; &pace; cluster. Among them
```

Especially as Corosync is written in capital C and pacemaker in lower case. With the entities, this inconsistency is gone.